searxng/docker-compose.yaml

```diff
@@ -0,0 +1,60 @@
+services:
+  redis:
+    image: ${SEARXNG_REDIS_IMAGE:-docker.io/valkey/valkey:latest}
+    command: valkey-server --save 30 1 --loglevel warning
+    restart: unless-stopped
+    volumes:
+      - valkey-data:/data
+    cap_drop:
+      - ALL
+    cap_add:
+      - SETGID
+      - SETUID
+      - DAC_OVERRIDE
```

Comment:
Did you test these or just copy from the whoami? If everything's working, great, but if not you might need to remove or change it.

Reply:
![image](https://github.com/user-attachments/assets/9e3caae7-bb3d-4858-8b9a-6eed58a9ec11)

Reply:
These were from the docker-compose.yml suggested by searxng. I haven't tested _without_ them, but everything works _with_ them.